mlflow/fastai.py

```diff
@@ -77,11 +77,9 @@ def get_default_conda_env(include_cloudpickle=False):
     pip_deps = None
     if include_cloudpickle:
         import cloudpickle
-
         pip_deps = ["cloudpickle=={}".format(cloudpickle.__version__)]
     return _mlflow_conda_env(
-        additional_conda_deps=["fastai={}".format(fastai.__version__)],
-        additional_pip_deps=pip_deps,
+        additional_pip_deps=["fastai={}".format(fastai.__version__)] + pip_deps,
```

Comment:
I think this line fails when `pip_deps` is None. Can we fix the code here to the following?

```python
    pip_deps = ["fastai={}".format(fastai.__version__)]
    if include_cloudpickle:
        import cloudpickle
        pip_deps.append("cloudpickle=={}".format(cloudpickle.__version__))

    return _mlflow_conda_env(
        additional_pip_deps=pip_deps,
        additional_conda_channels=None,
    )
```